meetings/2016-11-23.md

```diff
@@ -0,0 +1,272 @@
+providing a link to more info.
+
+@mhdawson: Would this mean we’d never add deprecation messages, except for
+security? Any deprecation message could lead to the same complaints.
+
+@trott: Not all messages are equal, Buffer comes up frequently.
+
+@mhdawson: This one is needed for end users to be aware of a security issue.
+Generally these warnings are mainly for developers.
+
+@addaleax: We’re unintentionally pushing people toward using `new Buffer` rather
+than the new APIs (`Buffer.alloc`).
+
+@Fishrock123: We should nudge people towards new and better APIs even if it
+causes them some pain. Is there any better way with the scale of the npm
+ecosystem?
+
+@mhdawson: New APIs don’t interfere with existing users; changing old APIs isn’t
+the same.
+
+@evanlucas: It’s our responsibility to work with the ecosystem to introduce
+these important changes. But not by pushing the pain to users.
+
+@jasnell: We should accept more pain in core to help users rather than deferring
+some pain to end users.
+
+If we want a new version of Buffer based on ES6 classes etc. we perhaps should
+start fresh, e.g. “Buffer2”, rather than breaking existing API.
+
+@Fishrock123: We’ll still have complaints about a new API.
+
+@jasnell: True, but they’re different complaints.
+
+@joshgav: Should we perhaps put deprecation warnings behind a flag by default?
+
+@jasnell has a PR which would allow this. Feedback is that users would not
+actually turn on the flag.
+
+@joshgav: Only need to turn it on every once in a while (@jasnell: e.g. in CI).
+
+@Fishrock123: If we don’t do the revert by next week than we should just drop
+it.
+
+@ChALkeR to get stats. If usage has gone down, don’t revert as it’s been
+effective. If not, might as well revert.
+
+@evanlucas: Stats don’t include private packages...
+
+@Fishrock123: But bigger concern is public modules which people depend on
+downstream.
+
+@ChALkeR: usage counts of public and private packages are quite different. Most
+users use public packages.
+
+@evanlucas: I think there are a lot of users using private packages.
+
+@ChALkeR: Yes, but these packages are not highly depended upon downstream.
+
+@evanlucas: If you have a (private) package used in many of your services which
+uses Buffer, it’s a pain.
+
+@mhdawson: Should we schedule a vote on “reverting the deprecation for Buffer
+without new”? Would be independent of future plans to e.g. hard-deprecate in
+v8.x.
+
+@addaleax: Schedule prior to next v7.x release.
+
+@Fishrock123: That will probably be next week, pending Node Interactive
+activities.
+
+* **Next Steps:** @evanlucas called for a vote and will set up an issue in the CTC repo.
+**[CTC#36](https://github.com/nodejs/CTC/issues/36)**
+
+---
+
+## Q/A on public channels
+
+### Q: What do you think about Yarn?
+
+@Fishrock123: Competition is good.
+
+@jasnell: Lots of promise, perf is phenomenal, will take a bit of time to
+mature.
+
+@mhdawson: Impressive, still some maturity to gain.
```

Comment:
make this:
@mhdawson: Its good to have options, it probably still needs some time to mature